build_deploy.sh

```diff
@@ -19,7 +19,8 @@ DOCKER_CONF="$PWD/.docker"
 mkdir -p "$DOCKER_CONF"
 docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
 docker --config="$DOCKER_CONF" login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
-docker --config="$DOCKER_CONF" build -t "${IMAGE}:${IMAGE_TAG}" .
+docker --config="$DOCKER_CONF" pull "${IMAGE}"
+docker --config="$DOCKER_CONF" build -t "${IMAGE}:${IMAGE_TAG}" --cache-from "${IMAGE}" --pull .
```

Comment:
or do we have `--no-cache` here? e.g. when there is a vulnerability in a rpm, we should probably install latest rpms for `:latest` image